### PR TITLE
(FM-7025) Remove reference to netdev_stdlib master from git repo from syncd gemfile

### PR DESCRIPTION
**Pull request by willmeek**

### CODE REVIEW

File: spec/spec_helper_acceptance.rb [hunk 1]
```diff
@@ -127,6 +127,7 @@ def device_model
         # install puppet-resource_api on to the server
         on(host, 'puppetserver gem install puppet-resource_api --no-ri --no-rdoc')
         on(host, 'puppet module install puppetlabs-resource_api')
+        on(host, 'puppet module install puppetlabs-netdev_stdlib')
```
Comment (tphoney) [inline spec/spec_helper_acceptance.rb]: https://github.com/willmeek/cisco_ios/blob/9c331d4e6737a58104cfe6e4491645af4a88650f/spec/spec_helper_acceptance.rb#L34 should do all the module installations needed.

Comment (willmeek) [reply spec/spec_helper_acceptance.rb]: Removed

File: .travis.yml [hunk 2]
```diff
@@ -5,21 +5,18 @@ language: ruby
...
   - bundle -v
-  - rm Gemfile.lock || true
+  - rm -f Gemfile.lock
   - gem update --system
   - gem update bundler
   - gem --version
   - bundle -v
 script:
-  - 'COVERAGE=yes bundle exec rake $CHECK'
+  - 'bundle exec rake $CHECK'
```
Comment (shermdog) [inline .travis.yml]: The PDK sync kills our coverage tests

Comment (willmeek) [reply .travis.yml]: I ended up down a PDK rabbit hole. I've pegged this back to just the immediate git get of the tagged release version.

Though I feel that re-enabling PDK should be done when I figure out how to enable coverage in travis.yml

**Pull request merged**